More info about bank reg numbers (#14)
BankAxept/bankaxept-onboarding-external-documentation: docs/getting_started.md

## Integration Guidelines

We validate the following information when registering an agreement:
* **Account Ownership**: We validate that the account number belongs to the customer.
* **Account Number**: We validate that the account number is valid and belongs to the customer.

> * **MCC**: We validate that the MCC is valid and exists in our database (we have fetched these from Baxbis).

Reviewer: ```suggestion
* **MCC**: We validate that the MCC is valid and supported```
